domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ClaimState.java

```diff
@@ -8,6 +8,7 @@
     CLOSED("closed"),
     SETTLED("settled"),
     READY_FOR_DIRECTIONS("readyForDirections"),
```

Comment:
Does this need changing to READY_FOR_LEGAL_ADVISOR_DIRECTIONS to be consistent with CaseEvent.java